woocommerce/woocommerce
# Update PNPM engine requirement to support PNPM 7.x

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/32885

We have PNPM locked to version 6.x and people are running into issues when they have PNPM 7.x installed. This PR updates the engines we have declared to support both 6.x and 7.x versions.

### How to test the changes in this Pull Request:

1. Delete the `pnpm-lock.yaml` file from repo root.
2. Install PNPM 7 globally by running `npm install -g pnpm@7`.
3. From this root repo, run `pnpm install`.
4. Ensure you are no longer seeing incompatible PNPM version and you're able to successfully install. Ignore the warnings of peer dependencies.
5. Try running the build script `pnpm -- turbo run build`. Ensure it is successfully built without errors.

### Other information:




### FOR PR REVIEWER ONLY:
